src/main/content/_assets/js/instance-common.js

```diff
@@ -197,9 +239,13 @@ function updateInstanceAdminView(adminMembersJson) {
         removeTeamMember(e.target);
     });
 
+    $(document).on("click", ".addUser-input-container button", e => {
+        addTeamMember(e.target);
+    });
+
```

Comment:
this probably doesn't need to be in a function that is called multiple times, it can result in multiple event listeners being added.

Probably best in the document load function. If the listener isn't getting set there, look into event delegation https://stackoverflow.com/a/15090957/1309377

check if the same problem exists for the other click events in this method